Source/WebCore/platform/graphics/skia/FontCacheSkia.cpp

```diff
@@ -29,7 +29,11 @@
 #include "Font.h"
 #include "FontDescription.h"
 #include "StyleFontSizeFunctions.h"
+#if defined(ANDROID)
```

Comment:
The [official answer](https://groups.google.com/g/android-ndk/c/cf9_f1SLXls/m/eI3UekPW4zEJ) seems to be that one should prefer `__ANDROID__` because it's always defined by the compiler, but in some places people have been using `ANDROID` as well... which explains why there's checks for both macros e.g. in ANGLE.

I would check both as well just in case:

```suggestion
#if defined(__ANDROID__) || defined(ANDROID)
```

Reply:
Fixed